### DISCUSSION TITLE
Phoenix Should Have Tabs To Go Back and Forth Between Projects

### DISCUSSION THREAD

**Discussion by treeguy1234** [phcode-dev/phoenix#1580] [Ideas]
I think it would be helpful if Phenix added tabs to the IDE so that we could switch between projects easily.

**Comment by abose** [comment]
Cool Idea! I created an issue to track: https://github.com/phcode-dev/phoenix/issues/1582
Might take a while to implement this, but can be done.

**Reply by treeguy1234** [reply]
Ok, Thanks!😀
